src/App.js

```diff
@@ -13,27 +20,65 @@ const onSelectedGenre = (genre) => {
 }
 
 const genres = [
-  'ALL',
-  'DOCUMENTARY',
-  'COMEDY',
-  'HORROR',
-  'CRIME',
-  'SUSPENSE'
+  'Action', 'Adventure', 'Science', 'Comedy', 'Drama'
 ]
 
+
+
 function App() {
+  const isDetailShowActive = true;
```

Comment:
What's the purpose of this constant?
I don't see being toggled somewhere else 😅